outreachy-round-13.md

```diff
@@ -0,0 +1,106 @@
+Zulip is a proud participant in [Outreachy](http://outreachy.org/) and
+is currently recruiting Outreachy interns for the upcoming Round
+13. We'll teach you and pay you to improve Zulip, full-time, between
+December 6, 2016 and March 6, 2017.
+
+If you're [in a group underrepresented in open source
+communities](https://wiki.gnome.org/Outreachy#Eligibility) and you're
+interested in contributing to Zulip during a paid apprenticeship,
+please apply by October 17th!
+
+# About us
+
+Zulip is a powerful, open source group chat application. Written in Python
+and using the Django framework, Zulip supports both private messaging and
+group chats via conversation streams.
+
+Zulip also supports fast search, drag-and-drop file uploads, image
+previews, group private messages, audible notifications, missed-message
+emails, desktop apps, and [much more](https://www.zulip.org).
+
+Zulip has gained a considerable amount of traction [since we released
+it as open source software in September
+2015](https://blogs.dropbox.com/tech/2015/09/open-sourcing-zulip-a-dropbox-hack-week-project/),
+and is gaining more users at a rapid pace -- every day, at least tens
+of thousands of people use Zulip. Your work on Zulip will have impact
+on the daily experiences of a huge number of people.
+
+As an organization, we value high-quality mentorship and making sure our
+product quality is extremely high -- you can except to learn a lot from
+disciplined code reviews by highly experienced engineers. Since Zulip is a
+group chat product, your Outreachy experience with the Zulip project will be
+highly interactive, with a real focus on teaching you the concepts and
+reasoning behind how Zulip is engineered and how to make it better.
+
+# Get to know us
+
+We have an easy-to-setup development environment, and a library of
+tasks that are great for first-time contributors
+([guide](https://github.com/zulip/zulip#contributing-to-zulip)).
+
+We'd love for you to drop by [our open livechat at
+[zulip.tabbott.net](https://zulip.tabbott.net/) or our [development
+mailing list](https://groups.google.com/forum/#!forum/zulip-devel),
+talk with us about [our
+roadmap](https://zulip.readthedocs.io/en/latest/roadmap.html) or your
+questions, and consider applying for an Outreachy internship with us!
```

Comment:
We should suggest a specific stream in zulip.tabbott.net and probably a mailing list that's a bit smaller than zulip-devel.  GSOC applicants resulted in 150 people emailing zulip-devel@ and resulted in a lot of people filtering the mailing list, and I'd like to avoid repeating that experience.